frontend/www/js/omegaup/components/group/Scoreboards.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <form class="form" @submit.prevent="onSubmitScoreboard">
```

Comment:
```suggestion
      <form class="form" @submit.prevent="onSubmit">
```

aquí estamos en el componente Scoreboard, así que no es necesario repetir esto en el nombre del evento.